### ISSUE TITLE
2.19 - Loaddem error because of deprecated method

### ISSUE THREAD

**Issue by camillemonchicourt** [GeotrekCE/Geotrek-admin#1756]
In Geotrek-admin 2.19, when launching ``loaddem`` command, I get an error (``AttributeError: type object 'BaseCommand' has no attribute 'option_list'``) : 

```
user@myserver:~/Geotrek-admin$ bin/django loaddem /home/user/mnt.tif
/home/user/Geotrek-admin/lib/eggs/psycopg2-2.7.4-py2.7-linux-x86_64.egg/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
Traceback (most recent call last):
  File "bin/django", line 125, in <module>
    sys.exit(djangorecipe.binscripts.manage('geotrek.settings.prod'))
  File "/home/user/Geotrek-admin/lib/eggs/djangorecipe-2.2.1-py2.7.egg/djangorecipe/binscripts.py", line 9, in manage
    management.execute_from_command_line(sys.argv)
  File "/home/user/Geotrek-admin/lib/eggs/Django-1.11.11-py2.7.egg/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/user/Geotrek-admin/lib/eggs/Django-1.11.11-py2.7.egg/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/user/Geotrek-admin/lib/eggs/Django-1.11.11-py2.7.egg/django/core/management/__init__.py", line 206, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/home/user/Geotrek-admin/lib/eggs/Django-1.11.11-py2.7.egg/django/core/management/__init__.py", line 40, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/user/Geotrek-admin/geotrek/altimetry/management/commands/loaddem.py", line 10, in <module>
    class Command(BaseCommand):
  File "/home/user/Geotrek-admin/geotrek/altimetry/management/commands/loaddem.py", line 17, in Command
    option_list = BaseCommand.option_list + (
AttributeError: type object 'BaseCommand' has no attribute 'option_list'
```

After reading https://docs.djangoproject.com/en/1.9/howto/custom-management-commands/#django.core.management.BaseCommand.option_list, it seems to be because the ``BaseCommand.option_list`` method is deprecated since Django version 1.8 while we are now in Django 1.11. We should now override the ``add_arguments()`` method to add custom arguments accepted by the ``loaddem`` command. 

See file : https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/altimetry/management/commands/loaddem.py#L17

But not sure how to replace the ``option_list`` by ``add_arguments()`` method.

**Comment by LePetitTim** [comment]
I will do it.
It's the same as the other function. This one is not tested actually.

**Comment by camillemonchicourt** [comment]
OK thanks !

**Comment by LePetitTim** [comment]
#1758

**Comment by camillemonchicourt** [comment]
Fix included in 2.21.0
